autobuild/build-cmake-common.sh

```diff
@@ -87,12 +88,12 @@ build_synfig_studio() {
     synfig_studio_make_cmd="make build_images"
     
     # Don't rerender the images if incremental is activated
-    if [ "$incremental_build" == "true" ]
+    if [ "$incremental_build" == "true" ] || [ ! "$rerender_images" == "true" ]
```

Comment:
I think here must be *AND* condition, something like:
if [ "$incremental_build" == "true" && "$rerender_images" != "true" ]

Reply:
> I think here must be _AND_ condition, something like:
> if [ "$incremental_build" == "true" && "$rerender_images" != "true" ]

Fixed in commit e98ea0d953007179faf1b4b14e3c609318136b06